wagtailio/static/js/components/feature-index.js

```diff
@@ -0,0 +1,82 @@
+class FeatureIndex {
+    static selector() {
+        return '[data-feature-index]';
+    }
+
+    constructor(node) {
+        this.node = node;
+        // 'all' checkbox input
+        this.allFilterCheckbox = this.node.querySelector('[data-feature-filter-all]');
+        // other checkbox inputs not including 'all'
+        this.filterCheckboxes = [...this.node.querySelectorAll('[data-feature-filter]')];
+        this.allFeatureGroups = [...this.node.querySelectorAll('[data-feature-group]')];
+        this.hiddenClass = 'is-hidden';
+
+        this.bindEvents();
+    }
+
+    bindEvents() {
+        // 'all' filter checkbox
+        this.allFilterCheckbox.addEventListener('change', (e) => {
+            if (e.target.checked) {
+                this.resetUI();
+            } else {
+                // Don't uncheck if it's the only one checked
+                this.allFilterCheckbox.checked = true;
+            }
+        })
+
+        // Other filter checkboxes not including 'all'
+        this.filterCheckboxes.forEach((filter) => {
+            filter.addEventListener('change', (e) => {
+                if (e.target.checked) {
+                    // Uncheck 'all' checkbox
+                    this.allFilterCheckbox.checked = false;
+
+                    this.handleShowGroup(e.target);
```

Comment:
I'm not sure this handles selecting from the "all" to another category. When selecting from "all" to "solutions" I can still see "Editing" which should be hidden, right?

Reply:
> When selecting from "all" to "solutions" I can still see "Editing" which should be hidden, right?

Definitely, you should only see 'Solutions' and not 'Editing'. I can't replicate on my end so can you send over a vid? Even better we can hop on a call if you have time : )